doc/governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,68 @@
+# Charter for the HoloViz Organization
+
+This is the organizational charter for the HoloViz Organization (the "Organization"). By adding their name to the [Steering Committee.md file](./STEERING-COMMITTEE.md), Steering Committee members agree as follows.
+
+## 1. Mission
+
+The mission of the Organization is to develop open-source solutions for a wide range of visualization problems in Python.
+
+## 2. Organization Steering Committee
+
+**2.1 Purpose**. The Steering Committee will be responsible for all technical oversight, project approval and oversight, policy oversight, and trademark management for the Organization.
+
+**2.2 Composition**. The Steering Committee voting members are listed in the steering-committee.md file in the repository. Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee. The Steering Committee will appoint a Chair responsible for organizing Steering Committee activity. There must be at least three individuals serving on the Committee at any time, but there is no upper limit on the number of Steering Committee members.
+
+## 3. Voting
+
+**3.1. Decision Making**. The Steering Committee will strive for all decisions to be made by consensus. While explicit agreement of the entire Steering Committee is preferred, it is not required for consensus. Rather, the Steering Committee will determine consensus based on their good faith consideration of a number of factors, including the dominant view of the Steering Committee and nature of support and objections. The Steering Committee will document evidence of consensus in accordance with these requirements. If consensus cannot be reached, the Steering Committee will make the decision by a vote.
+
+**3.2. Voting**. The Steering Committee Chair will call a vote with reasonable notice to the Steering Committee, setting out a discussion period and a separate voting period. Any discussion may be conducted in person or electronically by text, voice, or video. The discussion will be open to the public. In any vote, each voting representative will have one vote. Except as specifically noted elsewhere in this Charter, decisions by vote require a simple majority vote of all voting members.
+
+## 4. Termination of Membership
+
+In addition to the method set out in section 2.2, the membership of a Steering Committee member will terminate if any of the following occur:
+
+**4.1 Resignation**. Written notice of resignation to the Steering Committee.
+
+**4.2 Unreachable Member**. If a member is unresponsive at its listed handle for more than three months the Steering Committee may vote to remove the member.
```

Comment:
Can we clarify that such a vote would be on simple majority of the _remaining_ members? Otherwise if multiple members become unreachable we may be unable to have a binding vote. Similar issues apply elsewhere.

E.g. "the remaining reachable Steering Committee members may vote to remove the member".

Reply:
I've added text to both the org and project docs specifying that unless otherwise noted, decisions by vote require a simple majority vote of all reachable and vote-participating members. Let me know if this language suffices to specify those that are reachable and also have chosen to partake and cast an actual vote... that way votes can always be binding even if there are some people that abstain.

Reply:
Ok, sounds good!